articles/libraries/lock/v10/index.md

```diff
@@ -115,6 +115,14 @@ document.getElementById('btn-login').addEventListener('click', function() {
 });
 ```
 
+## Cross-Origin Authentication
+
+Using Lock within your application, rather than using the [Hosted Login Page](/hosted-pages/login), requires [cross-origin authentication](/cross-origin-authentication). In order to use embedded Lock via cross-origin authentication, you must do the following:
+
+* Set the [oidcconformant](/libraries/lock/v10/configuration#oidcconformant-boolean-) option to true
+* Set the [audience](/libraries/lock/v10/configuration#audience-string-) option
+* In the client settings area of the [Dashboard]($manage_url}), in the **Advanced Settings** menu, under the **OAuth** tab, turn on the OIDC Conformant and Cross Origin Authentication settings.
```

Comment:
These are fields so they should be formatted as bold: OIDC Conformant, Cross Origin Authentication

I'd also add a screenshots of these fields.